docs/userGuide/contentAuthoring.md

```diff
@@ -284,6 +285,92 @@ Front matter can also be included from a separate file, just like how content ca
 
 This will result in `index.md` having the title `Binary Search Tree` and the specified keywords in order for it to be looked up through the search bar.
 
+### Site Navigation
+
+A site navigation bar is a fixed menu on the left side of your page contents, that allows the viewer to navigate your site pages. 
+It has a fixed width of 300 pixels for its contents. 
+If you would like a site navigation bar for your site, you may create one easily using Markdown.
+
+- Start by creating a Markdown file (`.md`) in `_markbind/navigation`.
+    - All new sites created with `init` have a `site-nav.md` file created automatically for you as a template in `_markbind/navigation`.
+    - More than one navigation file can be created for different pages.
+
+- Author your navigation layout using Markdown's unordered list syntax as shown below.
+    - When you are using links (`[linkName](url)`), ensure that the url starts from the root directory <code>{<span></span>{baseUrl}}/</code> when you are referencing a page in your site. 
+    e.g. <code>{<span></span>{baseUrl}}/folder1/myPage.html</code>
+
+```html
+<!-- In _markbind/navigation/mySiteNav.md -->
+<markdown>
+* [Home :house:]({{baseUrl}}/index.html)
+* Dropdown title :pencil2: <!-- Nested list items will be placed inside a Dropdown menu -->
+  * [Dropdown link one](https://www.google.com/)
+  * Nested Dropdown title :triangular_ruler:
+    * [**Nested** Dropdown link one](https://www.google.com/)
+    * [**Nested** Dropdown link two](https://www.google.com/)
+  * [Dropdown link two](https://www.google.com/)
+* [==Third Link== :clipboard:](https://www.google.com/)
+* [Youtube](https://www.youtube.com/) <!-- Using a link as a Dropdown title will not render a Dropdown menu -->
+  * [The answer to everything in the universe](https://www.youtube.com/watch?v=dQw4w9WgXcQ)

```

Comment:
The other section has a YouTube URL as well. Let's standardize it by using the same URL:

```
http://www.youtube.com/watch?v=lJIrF4YjHfQ
```